weasyl/test/login/test_signin.py

```diff
@@ -18,7 +20,8 @@ def test_verify_login_record_is_updated():
     db = d.connect()
     db.add(sess)
     db.flush()
-    d.engine.execute("UPDATE login SET last_login = -1 WHERE userid = %(id)s", id=user_id)
+    time = datetime.fromtimestamp(-1, pytz.UTC)
```

Comment:
Ah, we should make this nullable instead of using `1969-12-31T23:59:59Z`.

```python
postgresql_using="CASE last_login WHEN -1 THEN NULL ELSE to_timestamp(last_login + 18000) END"
```
for the upgrade, I guess?

Reply:
Hmmm, should it be nullable though? We only seem to set it as -1 for this test to make sure it gets updated, the rest of the code doesn't seem to use any timestamps that low.
Theoretically, we could set it to any time prior to when the test is being run?

Reply:
Oh, I didn’t realize it was always set in site code. Thanks for explaining!

Reply:
Made a small change, to make it more clear what we are testing, anyway!